index.html

```diff
@@ -732,6 +788,7 @@ <h2>DisplayMediaStreamConstraints</h2>
 dictionary DisplayMediaStreamConstraints {
```

Comment:
This new member is not a constraint, so I think this is now officially an options object that among other things houses video and audio constraints: 
```suggestion
dictionary DisplayMediaStreamOptions {
```

Reply:
[getDisplayMedia](https://w3c.github.io/mediacapture-screen-share/#dom-mediadevices-getdisplaymedia) accepts [DisplayMediaStreamConstraints](https://w3c.github.io/mediacapture-screen-share/#dom-displaymediastreamconstraints), not `DisplayMediaStreamOptions`.